backend/app/models/project.rb

```diff
@@ -28,7 +28,7 @@ class Project < ApplicationRecord
     :progress_impact_tracking,
     :relevant_links
 
-  enum status: {draft: 0, published: 1, closed: 2}, _default: :draft
+  enum status: ProjectStatus::TYPES_WITH_CODE, _default: :draft
```

Comment:
in my local db project status is still a string field, and so the status filter doesn't work - is there a migration missing or did I mess sth up?